src/twisted/web/test/test_proxy.py

```diff
@@ -38,17 +38,21 @@ def _testRender(self, uri, expectedURI):
         # Clear the timeout if the tests failed
         self.addCleanup(channel.connectionLost, None)
 
-        channel.dataReceived(b"GET " +
-                             uri +
-                             b" HTTP/1.1\r\nAccept: text/html\r\n\r\n")
-
+        channel.dataReceived(
+            b'\r\n'.join([
+                b'GET %s HTTP/1.1' % uri,
```

Comment:
If we don't want to use `+` here (do we need the implicit `str()` call?) then we should probably change the RHS of the `%` operator to an explicit tuple

Reply:
I don't get you. You mean to change this line to `b'GET %s HTTP/1.1' % (uri,),` ?

Reply:
Oh. Then I'll leave this code unmodified